packages/bierzo-wallet/src/utils/test/persona.ts

```diff
@@ -9,7 +9,7 @@ const PASSWORD_CONFIRM_FIELD = "passwordConfirmInputField";
 const TERMS_ACCEPT_FIELD = "termsAcceptCheckboxField";
 
 async function clickCreatePersona(page: Page): Promise<void> {
-  await page.click("button:nth-of-type(2)");
+  await page.click("button:nth-of-type(1)");
```

Comment:
Can we get this button by ID or something similar, which is more stable? Especially in this case where the number of buttons are changing depending on the state of the extension